### DISCUSSION TITLE
I have a problem about the User-Agent

### DISCUSSION THREAD

**Discussion by huynhvuthienloc** [FlareSolverr/FlareSolverr#764] [Q&A]
Everything about FlareSolver is fine, but only the User-Agent.
I've tried to send the request with user-agent as "user-agent", but it does not work. However, the 'User-Agent' usually work. 
Only the 'User-Agent' works; the others like 'user-agent', 'User-agent', 'uSer-Agent',... do not work at all. 
I am using cURL PHP; does anyone has some problem with me?

**Comment by ngosang** [comment]
FlareSolverr gets the User-Agent from the Chrome/Chromium browser installed. You can not change it in any way. It will be detected by Cloudflare.
